docker_files/README.md

```diff
@@ -0,0 +1,12 @@
+# Docker Files
+
+Various Docker Files
+
+## File Mappings
+
+File Name     	    | Use                         | Docker Hub Link 
+------------------- | --------------------------- | ----------------------------------------------------------------------------
+scRNAseq_orchestra  | Orchestra WDL Docker Image  | [Docker Hub](https://hub.docker.com/r/singlecellportal/scrna-seq_orchestra/)
```

Comment:
Is this Docker file relevant for [`orchestra_methods.py`](https://github.com/broadinstitute/single_cell_portal/blob/b57311f33fbaca46e90f59350df63106469f818f/scripts/orchestra_methods.py)?  If so, perhaps note that in this README.

Reply:
So add a column like link to relevant files?